src/pages/signin/LoginForm/index.native.js

```diff
@@ -0,0 +1,48 @@
+import React, {useEffect, useRef} from 'react';
+import PropTypes from 'prop-types';
+import BaseLoginForm from './BaseLoginForm';
+import AppStateMonitor from '../../../libs/AppStateMonitor';
+
+const propTypes = {
+    /** Function used to scroll to the top of the page */
+    scrollPageToTop: PropTypes.func,
+};
+const defaultProps = {
+    scrollPageToTop: undefined,
+};
+
+function LoginForm(props) {
```

Comment:
NAB
```suggestion
function LoginForm({scrollPageToTop}) {
```

Reply:
The props are sent to the BaseLoginForm here: https://github.com/joh42/expensify-app/blob/2661fb84b0e3c4921aafa9fc3fe3e794be602d88/src/pages/signin/LoginForm/index.native.js#L38

But {scrollPageToTop, ...props} could work if that's preferred